.github/workflows/i18nify-js-validate.yml

```diff
@@ -30,6 +30,8 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: 20.3.1
+      - name: List files
```

Comment:
Why is this required ?

Reply:
my bad, was experimenting with gh actions.
Will remove.